src/NServiceBus.AwsLambda.SQS.AcceptanceTests/When_message_fails_processing.cs

```diff
@@ -46,8 +46,7 @@ public async Task Should_rethrow_when_disabling_error_queue()
 
             var exception = Assert.ThrowsAsync<Exception>(() => endpoint.Process(receivedMessages, null));
 
-            StringAssert.Contains("Failed to process message", exception.Message);
-            Assert.AreEqual("simulated exception", exception.InnerException.Message);
+            Assert.AreEqual("simulated exception", exception.Message);
```

Comment:
Not sure why this had to change

Reply:
Was able to track down the source of the bug here, and worked around it.